packages/scripts/docs/audit.md

```diff
@@ -6,11 +6,17 @@ Performs static dependency analysis
 
 The following checks are performed when this command is invoked.
 
-- ## audit-ci -h
+- ## audit-ci --config <.audit-ci.json path>
 
   ### Purpose
 
-  This tool will check for CVEs on any dependencies of the project. Any dependency with a **high** or **critical** level CVE will emit an error.
+  This tool will check for CVEs on any dependencies of the project. Any dependency with a **high** or **critical** level CVE that have not been [whitelisted](https://github.com/IBM/audit-ci#options) will emit an error.
+
+  When possible, use the following whitelist priority to minimize whitelist surface area:
+
+  1. `path-whitelist`
```

Comment:
1, 2, 3?

Reply:
Markdown supports auto-numbering, and generally speaking using all `1`s is the most appropriate way to write lists in Markdown. This way, if you add a bullet somewhere through the list, you don't have to rewrite the whole numbering.

Reply:
Ahh, gotcha; TIL